README.md

```diff
@@ -1,72 +1,41 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Project Name : Lasten-Ernest-portfolio website
 
-# Project Name
-
-> Description the project.
+> This is a mobile version a portfolio website belonging to Ernest Lasten as a web developer.
+> This website provides information about Ernest Lasten and his contact details.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML
+- CSS
 
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
 
+1. Clone this project into your computer and open the folder
+2. Right click on the index.html file and open with any browser
 
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.